packages/contracts-bedrock/justfile

```diff
@@ -15,24 +15,12 @@ dep-status:
 #                         BUILD                        #
 ########################################################
 
-# Checks that the correct version of Foundry is installed.
-check-foundry:
-  cd ../../ && ./ops/scripts/check-foundry.sh
-
-# Checks that semgrep is installed.
-check-semgrep:
-  cd ../../ && just check-semgrep
-
-# Checks that the correct versions of Foundry and semgrep are installed.
-check-dependencies:
-  just check-foundry && just check-semgrep
-
 # Core forge build command
 forge-build:
   forge build
 
 # Builds the contracts.
-build: check-dependencies lint-fix-no-fail forge-build interfaces-check-no-build
+build: lint-fix-no-fail forge-build interfaces-check-no-build
```

Comment:
Does `mise` have a check command? IMO it's useful to have a thing that validates that the versions are what we expect.

Reply:
`mise` will warn you on the command line if you're ever using the wrong version. You get a warning any time you open up a new folder. I figured that would be sufficient.